homeassistant/components/homekit/__init__.py

```diff
@@ -77,72 +77,61 @@ def get_accessory(hass, state, aid, config):
         _LOGGER.warning('The entitiy "%s" is not supported, since it '
                         'generates an invalid aid, please change it.',
                         state.entity_id)
-        return None
+        return
```

Comment:
Although they are the same, I would suggest we keep `return None` where we care / intend to use the return of a function

Reply:
I have again added it to `get_accessory` and `generate_aid`. In cases where it is an abort condition (like `HomeKit.start()`, `HomeKit.stop()`, `HomeAccessory.update_state_callback()`) I would prefer to stay with just `return`.